pom.xml

```diff
@@ -5,17 +5,16 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.18</version>
+        <version>3.43</version>
     </parent>
 
     <artifactId>external-workspace-manager</artifactId>
     <version>1.1.3-SNAPSHOT</version>
     <packaging>hpi</packaging>
 
     <properties>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.180</jenkins.version>
```

Comment:
Best target an LTS

Reply:
I agree that it is nice, but is it necessary?

Reply:
Why not leave it at 2.60.3?

Reply:
Addressed here: https://github.com/imaffe/external-workspace-manager-plugin/pull/4